### PR TITLE
Add native support for regex benchmark

### PR DESCRIPTION
**Pull request by jlb6740**

### CODE REVIEW

**Comment by jlb6740** [comment]
Depends on https://github.com/bytecodealliance/sightglass/pull/265

**Comment by jlb6740** [comment]
CI failure will go away once #265 is merged.

File: benchmarks/regex/Dockerfile.native [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#FROM ubuntu:20.04
+FROM rust:1.70
+#ENV DEBIAN_FRONTEND="noninteractive" TZ="America"
+#RUN apt update && apt-get install -y --no-install-recommends wget build-essential git ca-certificates
+
+# Install clang
+#RUN apt-get install -y --no-install-recommends clang
```
Comment (abrown) [inline benchmarks/regex/Dockerfile.native:7]: Remove?

File: benchmarks/regex/Dockerfile.native [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Copy in the `src` directory.
+ENV SRC=/usr/src/regex/
+WORKDIR $SRC
+ADD rust-benchmark rust-benchmark
+COPY sightglass.native.patch ./
+COPY build-native.sh .
+COPY libengine.so /usr/lib/
+RUN ls -l
```
Comment (abrown) [inline benchmarks/regex/Dockerfile.native:16]: Remove?

**Review by abrown — APPROVED** [review]

**Pull request merged**